leaflet-maplibre-gl.js

```diff
@@ -99,9 +99,23 @@
         getPaneName: function () {
             return this._map.getPane(this.options.pane) ? this.options.pane : 'tilePane'; 
         },
+
+        // Changes the zIndex of the layer.
+        setZIndex: function (zIndex) {
+          this.options.zIndex = zIndex;
+          this._updateZIndex();
+          return this;
+        },
+
+        _updateZIndex: function () {
+          if (this._container && this.options.zIndex !== undefined && this.options.zIndex !== null) {
```

Comment:
could be written with conditional chaining? 
```suggestion
          if (this._container && this.options?.zIndex) {
```